doc/tutorials/run_kata_containers.rst

```diff
@@ -0,0 +1,329 @@
+Virtual Machines (VMs) that feel and perform like containers, but provide
+the workload isolation and security advantages of VMs.
+
+Kata Code Repositories
+======================
+
+Kata containers code repositories: https://github.com/kata-containers/kata-containers
+
+.. list-table:: Kata Containers-developed components
+   :header-rows: 1
+   :widths: auto
+
+   * - Component name
+     - Description
+   * - **Agent**
+     - The `kata-agent <https://github.com/kata-containers/agent>`_ runs inside the virtual
+       machine and sets up the container environment.
+   * - **Proxy**
+     - The `kata-proxy <https://github.com/kata-containers/proxy>`_ is a process that runs
+       on the host and co-ordinates access to the agent running inside the virtual machine.
+   * - **Runtime**
+     - The `kata-runtime <https://github.com/kata-containers/runtime>`_ is usually invoked
+       by a container manager and provides high-level verbs to manage containers.
+   * - **Shim**
+     - The `kata-shim <https://github.com/kata-containers/shim>`_ is a process that runs on
+       the host. It acts as though it is the workload (which actually runs inside the virtual machine).
+       This shim is required to be compliant with the expectations of the
+       `OCI runtime specification <https://github.com/opencontainers/runtime-spec>`_.
+   * - **Packaging**
+     - We use the `packaging <https://github.com/kata-containers/packaging>`_ repository to
+       create packages for the `system components
+       <https://github.com/kata-containers/kata-containers#kata-containers-developed-components>`_
+       including `rootfs <https://github.com/kata-containers/kata-containers#os-builder>`_ and
+       `kernel <https://github.com/kata-containers/kata-containers#kernel>`_ images.
+   * - **OS builder**
+     - The `osbuilder <https://github.com/kata-containers/osbuilder>`_ tool can create a rootfs
+       and a "mini O/S" image. This image is used by the hypervisor to setup the environment
+       before switching to the workload.
+
+Pre-requisites
+**************
+
+.. _quick setup guide:
+   https://projectacrn.github.io/latest/tutorials/kbl-nuc-sdc.html#use-the-script-to-set-up-acrn-automatically
+
+.. _NUC7i7DNH:
+   https://ark.intel.com/content/www/us/en/ark/products/130393/intel-nuc-kit-nuc7i7dnhe.html
+
+#. Follow this `quick setup guide`_ to set up the Service VM based on Clear Linux 31670
+   onto your KBL NUC (`NUC7i7DNH`_).
+
+#. Enable the ``MAX_KATA_VM_NUM`` in ACRN hypervisor.
+
+   .. code-block:: none
+
+      $ cd acrn-hypervisor
+      $ git checkout v1.4
+      $ make menuconfig BOARD=nuc7i7dnb -C hypervisor
+      Modify ``Maximum number of Kata Containers in SOS`` to ``1`` and then Save and Quit.
+      $ make -C hypervisor
+      $ make -C misc/efi-stub HV_OBJDIR=$PWD/hypervisor/build EFI_OBJDIR=$PWD/hypervisor/build
+
+   Login Service VM, use the new hypervisor image:
+
+   .. code-block:: none
+
+      $ sudo mount /dev/sda1 /mnt
```

Comment:
Add a note here to mention to use `/dev/nvme0n1p1` in case of using an NVMe drive